easybuild/easyconfigs/m/mawk/mawk-1.3.4-20141206-GCC-4.9.2.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'mawk'
+version = '1.3.4-20141206'
+
+homepage = 'http://invisible-island.net/mawk/mawk.html'
+description = """mawk is an interpreter for the AWK Programming Language."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['ftp://invisible-island.net/mawk']
+
+sanity_check_paths = {
+    'files': ['bin/mawk'],
+    'dirs': ["bin"]
```

Comment:
I see this a lot in easyconfigs, and was already wondering...  Depending on the order of checking for files/dir, it might give more accurate errors.  However, the only way to find out is to dive into the framework, and there is only so much I want to know ;)